modules/microshift-lvms-system-requirements.adoc

```diff
@@ -11,15 +11,17 @@ Using LVMS in {microshift-short} requires the following system specifications.
 [id="lvms-volume-group-name_{context}"]
 == Volume group name
 
-The default integration of LVMS selects the default volume group (VG) dynamically. If there are no volume groups on the {microshift-short} host, LVMS is disabled.
+If the LVMS configuration is not present at `/etc/microshift/lvmd.yaml`, MicroShift will attempt to assign a default volume group dynamically using the `vgs` command.
```

Comment:
```suggestion
If you did not configure LVMS in an `lvmd.yaml` YAML file placed in the `/etc/microshift/` directory, {microshift-short} attempts to assign a default volume group (VG) dynamically by running the `vgs` command.
```
What are the conditions when it can and cannot auto-assign a VG? We should say. We are telling users here what the system requirements are to use LVMS. We don't want it disabled.